Tests/Client/RequestTests.cs

```diff
@@ -1,4 +1,10 @@
-﻿using System;
+﻿using Microsoft.AspNet.WebUtilities;
+using NUnit.Framework;
+using Pathoschild.Http.Client;
+using Pathoschild.Http.Client.Extensibility;
+using Pathoschild.Http.Client.Internal;
+using RichardSzalay.MockHttp;
+using System;
 using System.Collections.Generic;
```

Comment:
Minor style issues: `using System.*` should be sorted first, and the new unit tests don't match the current naming convention (e.g. `WithArgument_accepts_various_arg_types` should be `WithArgument_AcceptsVariousArgTypes`). Not a big deal though, I'll just standardise them when I accept the PR.

Reply:
You should think about adding a .editorconfig file to define your coding standards

Reply:
And also, think about using style analyzer to enforce you standards

Reply:
I use ReSharper to enforce standards, but I'll add an `.editorconfig` to share the style settings. :)